CDCgov/prime-reportstream
# 10740: elims mega message

This PR adds new translations for ELIMS, specifically handling specimen

Test Steps:
Using both sample messages in the linked issue
1. Update the `dev.DEV_ELIMS` receiver to be active
2. Send both messages from the `ignore.ignore-elr-elims` sender
3. Verify that the only discrepancies between the two are noted in the spreadsheets
https://cdc.sharepoint.com/:x:/r/teams/ReportStream/_layouts/15/Doc.aspx?sourcedoc=%7B2F962BDE-BCA1-4C16-B99E-7E8D31F3635D%7D&file=47_04608646_11024_MegaSpecimen.xlsx&action=default&mobileredirect=true
https://cdc.sharepoint.com/:x:/r/teams/ReportStream/_layouts/15/Doc.aspx?sourcedoc=%7B16FE9888-8743-466C-B78B-7F60D621129F%7D&file=47_32361_04608646_11034_MegaCase.xlsx&action=default&mobileredirect=true

## Changes
- Adds new HL7 -> FHIR extensions for specimen
- Updates the specimen translate schema to include all specimen values in the ELIMS message

## Checklist

### Testing

### Process

## Linked Issues
- #10740 

## To Be Done

## Specific Security-related subjects a reviewer should pay specific attention to
